contributors/guide/coding-conventions.md

```diff
@@ -16,119 +18,54 @@ description: |
-    - Libraries with no more appropriate home belong in new package
-subdirectories of pkg/util
-
-  - Avoid general utility packages. Packages called "util" are suspect. Instead,
-derive a name that describes your desired function. For example, the utility
-functions dealing with waiting for operations are in the "wait" package and
-include functionality like Poll. So the full name is wait.Poll
-
-  - All filenames should be lowercase
-
-  - Go source files and directories use underscores, not dashes
-    - Package directories should generally avoid using separators as much as
-possible (when packages are multiple words, they usually should be in nested
-subdirectories).
-
-  - Document directories and filenames should use dashes rather than underscores
-
-  - Contrived examples that illustrate system features belong in
-/docs/user-guide or /docs/admin, depending on whether it is a feature primarily
-intended for users that deploy applications or cluster administrators,
-respectively. Actual application examples belong in /examples.
-    - Examples should also illustrate [best practices for configuration and using the system](https://kubernetes.io/docs/concepts/configuration/overview/)
-
-  - Third-party code
-
-    - Go code for normal third-party dependencies is managed using
-[go modules](https://github.com/golang/go/wiki/Modules) and is described in the kubernetes
-[vendoring guide](/contributors/devel/sig-architecture/vendor.md)
-
-    - Other third-party code belongs in `/third_party`
-      - forked third party Go code goes in `/third_party/forked`
-      - forked _golang stdlib_ code goes in `/third_party/forked/golang`
-
-    - Third-party code must include licenses
-
-    - This includes modified third-party code and excerpts, as well
+  - Avoid package sprawl. Find an appropriate subdirectory for new packages. [See issue #4851](http://issues.k8s.io/4851) for discussion.
+    - Libraries with no appropriate home belong in new package subdirectories of `pkg/util`.
+  - Avoid general utility packages. Packages called "util" are suspect. Instead, derive a name that describes your desired function. For example, the utility functions dealing with waiting for operations are in the `wait` package and include functionality like `Poll`. The full name is `wait.Poll`.
+  - All filenames should be lowercase.
+  - Go source files and directories use underscores, not dashes.
+    - Package directories should generally avoid using separators as much as possible. When package names are multiple words, they usually should be in nested subdirectories.
+  - Document directories and filenames should use dashes rather than underscores.
+  - Contrived examples that illustrate system features belong in `/docs/user-guide` or `/docs/admin`, depending on whether it is a feature primarily intended for users that deploy applications or cluster administrators, respectively. Actual application examples belong in `examples`.
```

Comment:
```suggestion
```
This is actually no longer relevant, the docs directory in k/k have been deprecated.